bin/laminas-migration

```diff
@@ -11,7 +11,7 @@ namespace Laminas\Migration;
 
 use Symfony\Component\Console\Application;
 
-require __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/../../../autoload.php';
```

Comment:
I think, we need all cases here, something like we have here:
https://github.com/zfcampus/zf-development-mode/pull/37

Reply:
Agreed and added.

Reply:
There should be three cases, as in the PR I've linked:
```php
 if (file_exists($a = __DIR__ . '/../../../autoload.php')) {
     require $a;
 } elseif (file_exists($a = __DIR__ . '/../vendor/autoload.php')) {
     require $a;
 } elseif (file_exists($a = __DIR__ . '/../autoload.php')) {
     require $a;
 } else {
     // ...
```

Reply:
C&P provided example.